src/Analyzers/CSharp/Tests/RemoveUnnecessaryImports/RemoveUnnecessaryImportsTests.cs

```diff
@@ -186,22 +186,9 @@ public async Task TestGeneratedCode()
         var source = """
```

Comment:
Maybe add a `[WorkItem]` to the test to clarify, why no diagnostics there are expected